jujugui/static/gui/src/app/components/help/tour/tour.js

```diff
@@ -0,0 +1,130 @@
+/* Copyright (C) 2017 Canonical Ltd. */
+
+'use strict';
+
+const PropTypes = require('prop-types');
+const React = require('react');
+
+const Lightbox = require('../../lightbox/lightbox');
+const SvgIcon = require('../../svg-icon/svg-icon');
+
+class Tour extends React.PureComponent {
+  render() {
+    const tourPath = '/static/gui/build/app/assets/images/non-sprites/tour';
```

Comment:
this static path differs based on the environment, You'll have to pass in the static path as a prop.

Reply:
@hatched static path passed through as a prop returns `undefined` any idea why it wouldn't be defined?